### PR TITLE
[zh-cn]: create the translation of `navigator.userActivation` property

### PR DESCRIPTION
**Pull request by T34-active**

### CODE REVIEW

File: files/zh-cn/web/api/navigator/useractivation/index.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+使用 {{domxref("UserActivation.isActive")}} 检查用户是否正在与页面进行交互（{{Glossary("Transient activation", "瞬间激活")}}）。
+
+```js
+if (navigator.userActivation.isActive) {
+  // 继续请求播放媒体，例如
+}
+```
+
+### 检查用户是否曾经执行过操作
```
Comment (T34-active) [inline files/zh-cn/web/api/navigator/useractivation/index.md:28]:
```suggestion
### 检查用户曾经是否执行过操作
```

File: files/zh-cn/web/api/navigator/useractivation/index.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+使用 {{domxref("UserActivation.isActive")}} 检查用户是否正在与页面进行交互（{{Glossary("Transient activation", "瞬间激活")}}）。
+
+```js
+if (navigator.userActivation.isActive) {
+  // 继续请求播放媒体，例如
+}
+```
+
+### 检查用户曾经是否执行过操作
```
Comment (JasonLamv-t) [inline files/zh-cn/web/api/navigator/useractivation/index.md:28]:
```suggestion
### 检查是否曾执行过用户手势
```

File: files/zh-cn/web/api/navigator/useractivation/index.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+使用 {{domxref("UserActivation.hasBeenActive")}} 检查用户是否曾经与页面进行过交互（{{Glossary("Sticky activation", "粘性激活")}}）。
+
+```js
+if (navigator.userActivation.hasBeenActive) {
+  // 自动播放动画，例如
```
Comment (JasonLamv-t) [inline files/zh-cn/web/api/navigator/useractivation/index.md:34]:
```suggestion
  // 例如，继续自动播放动画
```

File: files/zh-cn/web/api/navigator/useractivation/index.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+## 浏览器兼容性
+
+{{Compat}}
+
+## 参见
+
+- {{domxref("UserActivation")}}
+- {{domxref("UserActivation.hasBeenActive")}}
+- {{domxref("UserActivation.isActive")}}
+- [基于用户激活的功能](/zh-CN/docs/Web/Security/User_activation)
```
Comment (JasonLamv-t) [inline files/zh-cn/web/api/navigator/useractivation/index.md:51]:
```suggestion
- [由用户激活控制的功能](/zh-CN/docs/Web/Security/User_activation)
```

**Review by JasonLamv-t — CHANGES_REQUESTED** [review]
Some suggestions for your reference.

**Review by JasonLamv-t — APPROVED** [review]
LGTM. Thank you! @T34-active

**Comment by T34-active** [comment]
Thank you! @JasonLamv-t

**Pull request merged**